_extras/Recipe.md

```diff
@@ -0,0 +1,75 @@
+
+We continue to be excited to learn from the innovations of our Instructor community! However, at this point we have also developed a better idea about the specific advantages and disadvantages of different practices. For those who do not wish to 'reinvent the wheel' but prefer to start with a recommended model, we offer an example structure for an online workshop that we expect to function well.
+
+Note that these recommendations are based on the use of Zoom, Etherpad, and Slack platforms and may require minor modifications to function on other platforms.
+
+## 1. Advance preparation
+### Workshop roles
+* **Instructors** (at least 2): take the lead on preparing the curriculum and leading instruction. 
+* **Supporting Instructors** (at least 1): take the lead on preparing the workshop mechanics, coordinating with helpers, and ensuring that everything is working smoothly. They may take on limited instruction and/or an instructional backup role if they choose. They play the role of "Zoom DJ" (below) whenever they are not teaching.
+### Schedule
+* A **4 half-day** schedule is ideal where practical. This is  helpful in combatting 'zoom fatigue.'
+* **Schedule plenty of breaks** and stick to them. Remember that breaks can be useful for troubleshooting, too.
+* See below with regard to timing challenges. If you are committed to covering the same content that you would in an in-person workshop, you may wish to extend your planned time.
+### Planning meetings
+* Plan at **least 2 meetings** before the workshop. 
+    * **Meeting 1 (well in advance)**: Establish workshop roles, who teaches what, delegate documentation & communications setup (below), plan asynchronous preparation.
+    * **Meeting 2 (closer to the date)**: Review workshop mechanics, backup planning, learner communications, assessment, and feedback review. Agree on content to cut when time runs short. Test video conferencing features.
+    
+For more detailed preparation goals, see our [checklist for online workshop preparation](https://carpentries.org/online-workshop-recommendations/#planning-your-workshop).
+### Back-channel communications
+* **Create a channel** or private conversation thread on The Carpentries Slack workspace (or other preferred tool). 
+* **Use your communication channel** to communicate about preparations. This will allow everyone on your instructional team to become familiar with the tool as well as improve your preparations.
+### Documentation
+* **Create an Etherpad** or other collaborative document, and add all vital information to be included in your workshop introduction for learners who arrive late. This should include:
+    * Workshop schedule (including the times of all breaks)
+    * Connection instructions
+    * What to do in the event of a connection failure
+    * How learners can ask questions (e.g. Zoom chat, raise hand, etc.)
+    * Other instructions related to workshop mechanics, including recommended window arrangement
```

Comment:
maybe also "and optionally logging in from another device to be able to see the screen from learners' perspective"